jdaviz/core/template_mixin.py

```diff
@@ -2289,6 +2291,21 @@ def __init__(self, plugin, *args, **kwargs):
         self._qtable = None
         super().__init__(plugin, 'Table', *args, **kwargs)
 
+    def default_value_for_column(self, colname=None, value=None):
+        if colname in self._default_values_by_colname:
+            return self._default_values_by_colname.get(colname)
+        if isinstance(value, (tuple, list)):
+            return [self.default_value_for_column(value=v) for v in value]
+        if isinstance(value, (float, int)):
+            return np.nan
+        if isinstance(value, str):
+            return ''
+        return None
+
+    @staticmethod
+    def _new_col_visible(colname):
+        return True
```

Comment:
Is this method effectively setting a default that gets overwritten [elsewhere](https://github.com/spacetelescope/jdaviz/pull/2093/files#diff-f8d8ca5f492ba5b9a6662f90b39d7e11ccff5ee72bce742257a154c13e540744R167)?

Reply:
exactly.  We can override the whole method to a lambda function if we want specific columns to be hidden by default.  If left alone, all added columns will be shown in the UI.